### PR TITLE
Update version

### PR DESCRIPTION
**Pull request by sakshamphul**
### Purpose/implementation Section
This PR is to update the version number to v0.90.0 and date of release 

#### What feature is being added or bug is being addressed?
N/A

#### What was your approach?
N/A

#### What GitHub issue does your pull request address?
N/A

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.
N/A
#### Which areas should receive a particularly close look?
N/A

#### Is there anything that you want to discuss further?
N/A

#### Documentation Checklist
N/A

- [ ] The function has examples to showcase the usage 
- [ ] The function is fully documented in the package using roxygen 
- [ ] Added a vignette

#### Package Checklist

- [ ] The dependencies required for the function in this pull request have been added to [DESCRIPTION](https://github.com/d3b-center/annoFuse/blob/master/DESCRIPTION) `Imports` section, and the corresponding `NAMESPACE` directives have been added/updated
- [ ] The files needed for the function to run are in `inst/extdata`
- [ ] The code was run through `styler`

### CODE REVIEW

**Review by jharenza — CHANGES_REQUESTED** [review]
Thanks for taking care of the release @sakshamphul! For this news document, can you retain the old release number with the features and just duplicate the headers (if applicable) to make a new release on top of that - this way, a user can look through the release history?

I realize we do not have this for all old releases and perhaps we need a separate ticket to update this file to add that info. 

Thanks!

File: NEWS.md
Comment (jharenza) [inline NEWS.md]: Eg - this would go above the previous release, like: 

```
# annoFuse v0.91.0 (released 2022-09-26)

## New features
.
.
.
.


# annoFuse v0.90.0 (released 2020-09-18)

## New features
.
.
.
```

File: NEWS.md
Comment (jharenza) [inline NEWS.md]:
```suggestion
* Add capability for annotation of custom fusion input file with minimal required columns
* Add custom type annotation that checks for required columns and maps input columns to output columns using config file
* Update fusion standardization function to accept custom fusion input that checks for few required columns 
* Config file is optional and can be use to map input columns with output columns
* Custom fusion input type was built on top of existing STAR-Fusion and Arriba fusion framework. 
* This version will continue to support both existing caller types with same implementation
```

**Comment by sickler-alex** [comment]
Will this create a tag too? I don't think I've ever done versions as a PR.

**Review by sickler-alex — APPROVED** [review]

**Review by migbro — APPROVED** [review]
Looks good!

**Comment by migbro** [comment]
> Will this create a tag too? I don't think I've ever done versions as a PR.

Um hmmm, probably not. If we really want to, we could, I think, delete the release once this is merged, and have it point at main

**Comment by migbro** [comment]
Also, I think @jharenza might have to re-review since she requested changes to let this pass

**Comment by sakshamphul** [comment]
@sickler-alex @migbro I have a tag attached to the latest release. I just have to update it to a specific commit once this is merged. @jharenza Can you review this PR and get me your approval final time before I merge?

**Review by jharenza — APPROVED** [review]

**Pull request merged**
